src/view/View.js

```diff
@@ -180,6 +180,12 @@ export default class View {
         permission = this.defaultReadPermission
       }
 
+      if (Reflect.has(this.query.fields, this.type)) {
+        if (this.query.fields[this.type].includes(attribute) === false) {
```

Comment:
no negates? :P

```suggestion
        if (!this.query.fields[this.type].includes(attribute)) {
```

Reply:
I do not use negation, I find the not operator easy to miss and make mistakes with and so prefer deliberate truthfulness comparisons